src/cargo/core/compiler/fingerprint/mod.rs

```diff
@@ -2206,6 +2419,22 @@ pub fn parse_rustc_dep_info(rustc_dep_info: &Path) -> CargoResult<RustcDepInfo>
                 None => None,
             };
             ret.env.push((unescape_env(env_var)?, env_val));
+        } else if let Some(rest) = line.strip_prefix("# checksum:") {
```

Comment:
As a side note: is it possible to combine a comment and the corresponding files into one line in rustc? It might make dep-info harder to read, though it is not meant to read by human, and combining them could potentially write less bytes. I dont know if it is really criticial in terms of performance.

```
src/lib.rs: src/someotherfile.rs # checksum:blake3=660e91cbd3 file_len:9
```

Reply:
See <https://github.com/rust-lang/rust/pull/126930#discussion_r1783822618>, which shows that my suggestion is not even better.